tests/unit/amo/components/TestContributeCard.js

```diff
@@ -134,7 +140,7 @@ describe(__filename, () => {
 
   it('sends a tracking event when the button is clicked', () => {
     const _tracking = createFakeTracking();
-    const contributionsUrl = 'some/url';
+    const contributionsUrl = { url: 'some/url', outgoing: 'some/other/url' };
```

Comment:
This ensures that we have a proper `contributionsUrl` to use below in `createInternalAddonWithLang`, but there's a problem later in the test. On line 154 we set `currentTarget: { href: contributionsUrl }`, but that's not going to be correct because `contributionsUrl` is not a URL but rather an object. The solution to this would be to replace that with `currentTarget: { href: contributionsUrl.outgoing }`, but the fact that this error doesn't cause the test to fail is a sign that we don't actually need `currentTarget` defined at all.

So I'd just replace that code with:

`const event = createFakeEvent();`

and see if the test still passes. If it does it means we never needed that code to begin with.

Reply:
https://github.com/mozilla/addons-frontend/pull/9960/commits/22475b17568b308fe64c49362ff08a61242f669a